packages/next/src/client/components/router-reducer/ppr-navigations.ts

```diff
@@ -242,6 +242,15 @@ export function updateCacheNodeOnNavigation(
     parallelRoutes: prefetchParallelRoutes,
   }
 
+  function newCacheNodeNeeded() {
+    if (needsDynamicRequest) {
+      return newCacheNode
+    } else {
+      return null
+    }
+  }
+
```

Comment:
Make this conditional statement simple and effective using ternary operator .
`return needsDynamicRequest ? newCacheNode : null`

And why do we need this as a separate function ?
I could see it was there in the node already

Reply:
Read:
#72224